lib/rack/lint.rb

```diff
@@ -513,129 +510,92 @@ def close(*args)
         end
       end
 
-      class HijackWrapper
-        extend Forwardable
-
-        REQUIRED_METHODS = [
-          :read, :write, :read_nonblock, :write_nonblock, :flush, :close,
```

Comment:
This has been split into full hijack which we agree should be an `IO` instance, and partial hijack which should follow the same interface as StreamWrapper.